crates/tower-cmd/src/deploy.rs

```diff
@@ -29,61 +35,69 @@ fn resolve_path(args: &ArgMatches) -> PathBuf {
 }
 
 pub async fn do_deploy(config: Config, args: &ArgMatches) {
-    // Determine the directory to build the package from
     let dir = resolve_path(args);
+    let auto_create = args.get_flag("auto-create");
+    if let Err(err) = deploy_from_dir(config, dir, auto_create).await {
+        match err {
+            crate::Error::ApiDeployError { source } => output::tower_error(source),
+            crate::Error::ApiDescribeAppError { source } => output::tower_error(source),
+            _ => eprintln!("Deploy error: {}", err),
```

Comment:
Is there actually a different type of error that can be returned here? We should quickly fix this in a follow-up PR.

Reply:
well hopefully not, but when i was trying to debug it, yes 😬